src/App.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './App.css';
+
+
+import {HashRouter, Switch, Route} from "react-router-dom"
+import SmartQuestionDetails from './view/SmartQuestionDetails';
+import SmartQuestionsList from './view/SmartQuestionsList';
+import SmartCreateQuestion from './view/SmartCreateQuestion';
+import SmartFilterQuestionsByTag from './view/SmartFilterQuestionsByTag';
+import SmartFilterQuestionsByTtitle from './view/SmartFilterQuestionsByTitle';
```

Comment:
Typo (Ttitle).

Reply:
thank you, this saves some of my future time